src/main/java/gov/nih/nci/icdc/IcdcEsFilter.java

```diff
@@ -325,8 +344,21 @@ private Map<String, Object> searchCases(Map<String, Object> params) throws IOExc
         return data;
     }
 
+    private List<String> fetchIdsFromResults(Map<String, Object> query, String endpoint, String idField, int size) throws IOException {
+        query.put("size", size);
+        Request request = new Request("GET", endpoint);
+        request.setJsonEntity(gson.toJson(query));
+        JsonObject result = esService.send(request);
+        JsonArray hitsArray = result.get("hits").getAsJsonObject().get("hits").getAsJsonArray();
+        List<String> idsArray = new ArrayList<>();
+        for (var id: hitsArray) {
```

Comment:
OPTIONAL CHANGE: I think it's best to avoid using `var` when explicit typing is available in a language. I'll leave it up to you but I would recommend changing this to an explicit type even though there is very little impact in this usage